script/Ciel/Day 11/11_01_CIEL11_2_1.txt

```diff
@@ -204,7 +204,7 @@ The creatures humans prey on may be different,
 [sha:feaed1e59d8ec40ec6eae4eb5a79ce97161b966f]{
 -- Page 108, Offset 29364.
 -- であれば、<殺|ア><人|レ>を悪と<罵|ののし>る事こそ<傲慢|ごうまん>だ。
-With that in mind, do you not think it arrogant to deem <homicide|p r e d a t i o n   > evil?
+With that in mind, do you not think it's arrogant to deem <homicide|p r e d a t i o n    > evil?
```

Comment:
Revert, dramatic phrasing (Roa love the dramatic phrasing)

Reply:
these dramatic phrasings are bodying me